uctMaths/competition/compadmin.py

```diff
@@ -630,6 +635,34 @@ def export_awards(request):
     output_workbook.save(response)
     return response
 
+def makeCertificates(students, assigned_school):
+    
+    certPath = "../Certificates/"
+
+    if len(students) > 0:
+        awardCerts = {"G": "goldTemplate.docx", "M": "meritTemplate.docx", None: "participationTemplate.docx", "MOX": "meritOxfordTemplate.docx", "OX": "oxfordTemplate"}
+        schoolname = ''.join([i if ord(i) < 128 else '' for i in repr(assigned_school)])
+        schoolname = schoolname[8:-1].strip()
+        tmpDir = tempfile.mkdtemp()
```

Comment:
You should delete this directory. I recommend wrapping the rest of the code in a try finally block e.g.

``` python
tmpDir = tempfile.mkdtemp()
try:
    # <<The rest of the code>>
finally:
    shutil.rmtree(tmpDir)